### PR TITLE
Adding MongoDB Event Table Implementation

### PR DESCRIPTION
**Pull request by niveathika**

### CODE REVIEW

File: component/pom.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (dnwick) [inline component/pom.xml]: update the license year

**Review by dnwick — COMMENTED** [review]
approved

File: component/pom.xml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Private-Package>
+                            com.mongodb.*;
+                            org.bson.*;
+                        </Private-Package>
+                        <Export-Package>
+                            org.wso2.siddhi.extension.store.mongodb.*
+                        </Export-Package>
+                        <Import-Package>
+                            *;resolution:=optional
```
Comment (ramindu90) [inline component/pom.xml:78]: do we know the exact packages to import?

File: component/pom.xml (hunk 1 above)
Comment (niveathika) [reply component/pom.xml]: Fixed in 640591f827900ecfd1799c5de392e31c56bb9410

File: component/pom.xml (hunk 2 above)
Comment (niveathika) [reply component/pom.xml:78]: Yes.  Fixed in 92a251b

**Pull request merged**